stages/org.osbuild.grub2

```diff
@@ -110,17 +120,19 @@ def write_grub_cfg(tree, path):
                   "blscfg\n")
 
 
-def write_grub_cfg_redirect(tree, path):
+def write_grub_cfg_redirect(tree, path, separate_boot):
     """Write a grub config pointing to the other cfg"""
     print("hybrid boot support enabled. Writing alias grub config")
+    root = "/" if separate_boot else "/boot"
     with open(os.path.join(tree, path), "w") as cfg:
-        cfg.write("search --no-floppy --set prefix --file /boot/grub2/grub.cfg\n"
-                  "set prefix=($prefix)/boot/grub2\n"
+        cfg.write(f"search --no-floppy --set prefix --file {root}/grub2/grub.cfg\n"
```

Comment:
When root is not separate, this will resolve to `//grub2/grub.cfg`, that should be equivalent, but maybe just move the `/` to make it pretty?

Reply:
Good catch, fixed!